pkgs/tools/security/cliam/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "cliam";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "securisec";
+    repo = pname;
+    rev = version;
+    hash = "sha256-bq7u6pknokyY4WwO1qMYPuY86UZlDgeYEa1AJpk8d+4=";
+  };
+
+  vendorSha256 = "sha256-aGBA97EvIUv9myqcrtltiVxh1/0VtrQy2j9GU6r197g=";
+
+  ldflags = [
+    "-s -w"
+    "-X github.com/securisec/cliam/cli/version.Version=${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/$pname
+  '';
+
```

cliam looks like it provides completions according to the readme. might be worth adding something like this:

```suggestion
  postInstall = ''
    installShellCompletion --cmd cliam \
      --bash <($out/bin/cliam completion bash) \
      --fish <($out/bin/cliam completion fish) \
      --zsh <($out/bin/cliam completion zsh)
  '';
```

---

Added, thanks.